enterprise_subsidy/apps/fulfillment/api.py

```diff
@@ -84,6 +87,12 @@ def _get_geag_variant_id(self, transaction):
         ent_uuid = self._get_enterprise_customer_uuid(transaction)
         return ContentMetadataApi().get_geag_variant_id(ent_uuid, transaction.content_key)
```

Comment:
if we like the pattern of using the client from the subsidy, we can do a similar treatment here